src/app/components/layout/footer/footer.component.html

```diff
@@ -1,7 +1,10 @@
 <div>
-  <mat-toolbar
-    class="footer"
+  <mat-toolbar class="footer"
     [color]="footerColor">
     {{'FOOTER.PUT' | translate}}
   </mat-toolbar>
+  <mat-toolbar class="footer"
+               [color]="footerColor">
+    <a style="color: gray" href="http://www.put.poznan.pl">www.put.poznan.pl</a>
```

Comment:
`style="color: gray" `
setting style in html? Why not is css?

Reply:
Cause Im lazy, hello?